.flake8

```diff
@@ -2,3 +2,5 @@
 ignore = \
     E501, \ # line too long (black fixes long lines, except for long strings which may benefit from being long)
     W503    # line break before binary operator (black disagrees)
+exclude =
```

Comment:
If we're using a relatively modern flake8 then we can use exclude-extends to no wipe out the default exclude.

Reply:
yup, this is in the next PR https://github.com/ebmdatalab/ebmbot/pull/254